src/node/blockstorage.cpp

```diff
@@ -156,14 +156,42 @@ bool CBlockIndexWorkComparator::operator()(const CBlockIndex* pa, const CBlockIn
     if (pa->nChainWork > pb->nChainWork) return false;
     if (pa->nChainWork < pb->nChainWork) return true;
 
-    // ... then by earliest time received, ...
```

Comment:
If someone wants to pick this up:
"// ... then by earliest time received, ..."
in the existing code is not correct and should be something like "// ... then by earliest time activable, ...", or a more verbose explanation.